{
  "vid": "8719b935-8bae-41ad-92ba-3c826f651219",
  "topic": "python 2.7 -- multiple vulnerabilities",
  "description": "\n\t\u003cp\u003epython release notes:\u003c/p\u003e\n\t\u003cblockquote cite=\"https://github.com/python/cpython/blob/2.7/Misc/NEWS.d/2.7.15rc1.rst\"\u003e\n\t  \u003cp\u003eMultiple vulnerabilities has been fixed in this release. Please refer to the CVE list for details.\u003c/p\u003e\n\t\u003c/blockquote\u003e\n      ",
  "affects": [
    {
      "name": "python27",
      "range": [
        {
          "lt": "2.7.15"
        }
      ]
    }
  ],
  "dates": {
    "discovery": "2018-05-01T00:00:00Z",
    "entry": "2018-05-05T00:00:00Z"
  },
  "references": [
    {
      "source": "URL",
      "text": "https://github.com/python/cpython/blob/2.7/Misc/NEWS.d/2.7.15rc1.rst"
    },
    {
      "source": "URL",
      "text": "https://github.com/python/cpython/blob/2.7/Misc/NEWS.d/2.7.15.rst"
    },
    {
      "source": "CVE",
      "text": "CVE-2012-0876"
    },
    {
      "source": "CVE",
      "text": "CVE-2016-0718"
    },
    {
      "source": "CVE",
      "text": "CVE-2016-4472"
    },
    {
      "source": "CVE",
      "text": "CVE-2016-9063"
    },
    {
      "source": "CVE",
      "text": "CVE-2017-9233"
    },
    {
      "source": "CVE",
      "text": "CVE-2018-1060"
    },
    {
      "source": "CVE",
      "text": "CVE-2018-1061"
    }
  ]
}
